app/src/main/java/co/develhope/chooseyouowncocktail_g2/adapter/CustomListAdapter.kt

```diff
@@ -7,12 +7,15 @@ import android.app.Activity
 import android.view.View
 import android.view.ViewGroup
 import android.widget.*
+import co.develhope.chooseyouowncocktail_g2.DrinkAction
 import co.develhope.chooseyouowncocktail_g2.model.Beer
 import com.squareup.picasso.Picasso
 
-class CustomListAdapter(private val context: Activity,
-                        private val beer: List<Beer>,
-                        val action: (DrinkAction) -> Unit) :
+class CustomListAdapter(
+    private val context: Activity,
```

Comment:
passare l'activity in un adapter è una bad practice, non serve a nulla. il contesto lo hai prescindere per ci sono le views

Reply:
Non ho fixato perchè lo ha già fatto Francesco Pedullà in un altro branch